examples/nlp/token_classification/token_classification_train.py

```diff
@@ -101,15 +102,15 @@
 
 @hydra_runner(config_path="conf", config_name="token_classification_config")
 def main(cfg: DictConfig) -> None:
-    trainer = pl.Trainer(**cfg.trainer)
+    trainer = pl.Trainer(plugins=[NLPDDPPlugin()], **cfg.trainer)
```

Comment:
same question here - how does this interact with non-model-parallel

Reply:
Non-model parallel case works as normal with NLPDDPPLugin.